index.html

```diff
@@ -529,28 +529,58 @@ <h2>
         Examples
       </h2>
       <p>
-        This example shows the current screen orientation to the console every
-        time the screen orientation state changes.
+        In this example for a mobile device, clicking the landscape button
+        makes a request to go fullscreen and then lock to landscape
+        orientation. Once it is in landscape, pressing the unlock button
+        reverts the screen orientation back to default (portrait). Pressing the
+        button labelled "portrait" will also change it back to <a>portrait</a>.
+      </p>
+      <p>
+        The log shows the change in orientation type and angle, and the type of
+        event.
       </p>
       <pre class='example html'>
 &lt;script&gt;
-  var show = function() {
-     console.log("Orientation type is " + screen.orientation.type);
-     console.log("Orientation angle is " + screen.orientation.angle);
-  }
+async function fullScreenCheck() {
+  if(document.fullscreenElement === null) {
+      await document.documentElement.requestFullscreen();
+  } 
+}
+
+async function lockLandscape(){
+  await fullScreenCheck() 
+  await screen.orientation.lock('landscape');
+}
+
+async function unlock() {
+  await fullScreenCheck();
+  await screen.orientation.unlock();
+}
+
+async function portrait(){
+  await fullScreenCheck();
+  await screen.orientation.lock('portrait');
+}
 
-  screen.orientation.addEventListener("change", show);
-  window.onload = show;
+const { type, angle } = screen.orientation
```

Comment:
This could be inside the show() function, then you can just:

```
const { type, angle } = screen.orientation
console.log("Orientation type is " + type);
console.log("Orientation angle is " + angle);
```

Reply:
Might be less redundant to do:

```JS
const { type, angle } = screen.orientation
console.log(`Orientation type Is "${type}", angle is ${angle}.`);
```